cfgov/v1/models/home_page.py

```diff
@@ -289,8 +68,21 @@ class HomePage(CFGOVPage):
     # Tab handler interface
     edit_handler = TabbedInterface([
         ObjectList(content_panels, heading='General Content'),
-        ObjectList(CFGOVPage.sidefoot_panels, heading='Sidebar'),
         ObjectList(CFGOVPage.settings_panels, heading='Configuration'),
+        ObjectList([
+            InlinePanel(
+                'carousel_items', min_num=4, max_num=4, label="Carousel Item"
+            ),
+        ], heading='Carousel'),
+        ObjectList([
+            InlinePanel(
+                'info_units', min_num=6, max_num=6, label="Info Unit"
+            ),
+        ], heading='Info Units'),
+        ObjectList([
+            FieldPanel('card_heading'),
+            InlinePanel('cards', min_num=3, max_num=3, label="Card"),
+        ], heading='Cards')
```

Comment:
One more comment, I guess. while the Carousel would need a lot of work to support something other than exactly four items, perhaps we should be more flexible on the info units and cards? /cc @sonnakim

Reply:
I thought of this; it'd be nice, for example, to enforce that the number of cards or info units is always divisible by 3. I'm not sure that's easy to do in Wagtail because of how it relies on Django's built-in [formset validation](https://docs.djangoproject.com/en/2.2/topics/forms/formsets/#validating-the-number-of-forms-in-a-formset).

Reply:
I don't even know that it's necessary to make it divisible by 3. Allow anything from some reasonable minimum (2?) to some reasonable maximum (6?) We could use CSS to change the layout to 2-column if it's an even number, which would eliminate the worst-looking case of 4 items in a 3-column grid.